app/code/Magento/NewsletterApi/Api/NewsletterSubscriptionManagementInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\NewsletterApi\Api\Data;
+
+/**
+ * Newsletter Subscription Management Interface
+ *
+ * @api
+ */
+interface NewsletterSubscriptionManagementInterface
+{
+    /**
+     * @param \Magento\NewsletterApi\Api\Data\NewsletterSubscriptionInterface $newsletterSubscription
+     * @return \Magento\NewsletterApi\Api\Data\NewsletterSubscriptionInterface
```

Comment:
["command"](https://martinfowler.com/bliki/CQRS.html) in terms of CQRS
suppose to return void or ID of newly created entity